client/state/ui/guided-tours/selectors.js

```diff
@@ -2,16 +2,165 @@
  * External dependencies
  */
 import get from 'lodash/get';
```

Comment:
We can do this now :D https://github.com/Automattic/wp-calypso/pull/6539/files#diff-4aaad6137f1bc91a0f39e5b5e3ddb478R6